### PR TITLE
Added Discovery 6.0 documentation

### PR DESCRIPTION
**Pull request by lzap**
SSIA

### CODE REVIEW

File: plugins/foreman_discovery/6.0/index.md [hunk 1]
```diff
@@ -0,0 +1,1183 @@
...
+  </tr>
+  <tr>
+    <td>= 1.10</td>
+    <td>4.1.1</td>
+    <td>1.0.3</td>
+    <td>3.0</td>
+    <td>0.0.2</td>
+  </tr>
+  <tr>
+    <td>= 1.12</td>
```
Comment (domcleal) [inline plugins/foreman_discovery/6.0/index.md]: Should this be _below_ 1.11? Shouldn't it use the page variables?

Comment (lzap) [inline plugins/foreman_discovery/6.0/index.md]: > Should this be _below_ 1.11? Shouldn't it use the page variables?

Oh yeah, I actually swapped them! Nice catch, thanks.

## 

Later,
 Lukas #lzap Zapletal

**Comment by dLobatog** [comment]
Merged as 4e4482050a473be12248cef8b095a271fd946337, thanks @lzap!

**Comment by dLobatog** [comment]
0dd27e3b1daf99d91dbe3fdeb97d0473cff8673c rather, I rebased your 2 commits

**Pull request closed without merging**